feat(data-viz): highcharts a11y (#4053)
twilio-labs/paste: packages/paste-website/src/pages/foundations/data-visualization/index.mdx

The easiest way to use the data visualization tokens in your charts is to use `usePasteHighchartsTheme` from the [data visualization library package](/core/libraries/data-visualization) with [Highcharts](https://www.highcharts.com/). The `usePasteHighchartsTheme` hook takes an object of Highchart configurations and returns a new object with Paste colors and fonts.

> We strongly advise that you include the [Highcharts Accessibility module](/foundations/data-visualization#adding-highcharts-accessibility-module) to ensure that your charts are accessible to all UserMessage.

Reviewer: question: What's UserMessage?

Author: Sorry that's my copilot going crazy chaging user to a variable. fixed in upcoming commit

> We strongly advise that you include the [Highcharts Accessibility module](/foundations/data-visualization#adding-highcharts-accessibility-module) to ensure that your charts are accessible to all UserMessage.

Reviewer: ```suggestion
Be sure to include the [Highcharts Accessibility module](/foundations/data-visualization#adding-highcharts-accessibility-module) to ensure that your charts are accessible to all UserMessage.
```

<LineChartCaption />

### Adding Highcharts Accessibility module

> Highcharts by default does no include it's [accessability](https://www.highcharts.com/docs/accessibility/accessibility-module) module. Every Highcarts license includes the Accessibility module which adheres to the WCAG 2.1 standard. You will not need to do anything other than add the module to Highcarts to gain enhanced keyboard navigation and screen reader functionality.

Reviewer: ```suggestion
Highcharts by default does not include its [Accessibility](https://www.highcharts.com/docs/accessibility/accessibility-module) module. Every Highcharts license includes the Accessibility module which adheres to the WCAG 2.1 standard and adds enhanced keyboard navigation and screen reader functionality.
```

A helper function `applyPasteHighchartsModules` is available from the [data visualization library package](/core/libraries/data-visualization#applypastehighchartsmodules), which applies additional modules to Highcharts. This function should be called at the component level where the Highcharts namespace is passed to HighchartsReact.

```jsx
import Highcharts from "highcharts";
import HighchartsReact from "highcharts-react-official";
import HighchartsAccessibilityModule from "highcharts/modules/accessibility";
import HighchartsExportingModule from "highcharts/modules/exporting";
import { applyPasteHighchartsModules } from "@twilio-paste/data-visualization-library";

applyPasteHighchartsModules(Highcharts, HighchartsAccessibilityModule, HighchartsExportingModule);

const LineChartExample = () => {
  return <HighchartsReact highcharts={Highcharts} />;
};
```

</content>

> </contentwrapper>applyPasteHighchartsModules,

Reviewer: issue: loose text here

Author: fixed in upcoming commit
